app/presenters/api/integrations_show_presenter.rb

```diff
@@ -53,29 +53,12 @@ def environments_have_same_config?
     end
   end
 
-  def apiap?
-    provider_can_use? :api_as_product
-  end
-
   def apicast_config_ready?
-    if apiap?
-      proxy.service.backend_apis.any?
-    else
-      any_sandbox_configs?
-    end
+    proxy.service.backend_apis.any?
```

Comment:
I will accept this for now because it is not your fault and the PR would get too big if I start complaining about these things 😂